homeassistant/components/zamg/config_flow.py

```diff
@@ -64,22 +70,21 @@ async def async_step_user(
         try:
             self._client.set_default_station(station_id)
             await self._client.update()
-        except (ValueError, TypeError) as err:
+        except (ZamgApiError, ZamgNoDataError) as err:
             LOGGER.error("Config_flow: Received error from ZAMG: %s", err)
             errors["base"] = "cannot_connect"
```

Comment:
If we're just going to abort we don't need to set any error strings. Those are only needed if we show a form.